cuda_core/cuda/core/experimental/include/utility.hpp

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2025, NVIDIA CORPORATION & AFFILIATES. ALL RIGHTS RESERVED.
+//
+// SPDX-License-Identifier: LicenseRef-NVIDIA-SOFTWARE-LICENSE
+
+#pragma once
+
+#include <type_traits>
+
+// In cuda.bindings 12.8, the private member name was renamed from "_ptr" to "_pvt_ptr".
```

Comment:
@vzhurba01 do you recall what prompted this change when updating to 12.8?

Reply:
We're using a private member? Do we own the code with the private member?

Reply:
Yes we own everything, they are autogen'd, for example for `CUstream`:
https://github.com/NVIDIA/cuda-python/blob/02a41789641dec6ab5fa4c07ed12ad8600aa5be4/cuda_bindings/cuda/bindings/driver.pxd.in#L185-L197
hence I feel SFINAE over all possibilities is OK. I think if we accidentally change the codegen in the future, the added Cython test should be able to catch it.